Issue#2 register screen mockup

Namoku opened the pull request:
# Description
What did you implemented/Why did you implemented this:
 - Add form design
 - Add buttons
 - Add overlay
 - Add checkboxes

  ## Screenshots 
![image](https://user-images.githubusercontent.com/31052826/146623503-caba6544-e520-454b-9876-8b63ec589523.png)

Darkboy5120 committed:
Register mockup 0.1

- Componentes arent abstracted correctly with atomic structure right now, that will happend when we a more complete register mock
- Almos everything in the image is complete (form, error msg, checkbox, submit buttons, already account text), we just need install icons (i am unable to do it, windows error o something like that)
- Buttons and checkbox are using useState,. Buttons update their style and the disabled prop, checkbox updates the icon
- The loading modal isnt implemented yet, its the only thing left and maybe tinker some elements units (padding, height, colors, etc)

Co-Authored-By: thisisofficial <thisisofficiallyofficial@gmail.com>

Namoku committed:
Fix icons not showing in sign up form.

Namoku committed:
Add loading modal to register screen, and google icon in sign up button.

Co-authored-by: Darkboy5120 <hmaldoando0@ucol.mx>
Co-Authored-By: thisisofficial <thisisofficiallyofficial@gmail.com>

Namoku committed:
Change file structure in atoms, overlay and buttons
- Remove deprecated buttons (SignButton, CustomModal)
- Add colors
- Fixed input field

Co-authored-by: Darkboy5120 <hmaldoando0@ucol.mx>

Namoku committed:
Refactor for fix issue from Codeclimate

Namoku committed:
Refactor for fix issue from Codeclimate

Namoku committed:
Merge branch 'issue#2Register-screen-mockup' of github.com:BC-MAY-21-RN/kata-05-reservacion-de-vuelos-k5-n21-t2 into issue#2Register-screen-mockup

YilKanda commented on ios/kata05/Info.plist:
```diff
@@ -51,5 +51,24 @@
 	<false/>
+	<key>UIAppFonts</key>
+	<array>
+		<string>AntDesign.ttf</string>
+		<string>Entypo.ttf</string>
+		<string>EvilIcons.ttf</string>
+		<string>Feather.ttf</string>
+		<string>FontAwesome.ttf</string>
+		<string>FontAwesome5_Brands.ttf</string>
+		<string>FontAwesome5_Regular.ttf</string>
+		<string>FontAwesome5_Solid.ttf</string>
+		<string>Fontisto.ttf</string>
+		<string>Foundation.ttf</string>
+		<string>Ionicons.ttf</string>
+		<string>MaterialCommunityIcons.ttf</string>
+		<string>MaterialIcons.ttf</string>
+		<string>Octicons.ttf</string>
+		<string>SimpleLineIcons.ttf</string>
+		<string>Zocial.ttf</string>
```
Why are you adding multiple fonts?

Darkboy5120 replied:
We miss that detail, we have just fixed!, we are focusing in android version right now, but we will use the ionicons ttf, in both versions in order to use only one, as you said, we dont want to confuse the user, thanks for the feedback

YilKanda reviewed (changes requested):
My observation in this PR is that you are installing too many typefaces. It is advisable to have a guideline of typefaces to use. Example, I leave you the link of [FontJoy](https://fontjoy.com/), where you can make a combination of 3 fonts. But remember that the combination of many typographic fonts only incites confusion and is not intuitive for Internet users, in addition the use of a certain font is also essential since each one has a personality and transmits a message.

I imagine all these fonts are added by icons, but do you really take up all the icons from those fonts?

Darkboy5120 committed:
Ionicons as unique ttf

Set Ionicons as unique ttf in boths, android and ios

Co-Authored-By: Nando San Miguel Lopez <contact@namoku.xyz>
Co-Authored-By: thisisofficial <thisisofficiallyofficial@gmail.com>

Darkboy5120 reviewed (approved):
Thanks for the feedback!, we have just fixed that, we will take fonts in consideration from now, thanks again :)

Darkboy5120 reviewed (dismissed):
Thanks to you Yill for your feedback, we will take in consideration from now using only one or very few fonts

YilKanda commented on src/screens/LoginScreen.js:
```diff
@@ -0,0 +1,85 @@
+    backgroundColor: 'rgb(50, 50, 50)',
+  },
+  PrimaryColor: {
+    backgroundColor: '#5566dd',
+    borderRadius: 15,
+    alignSelf: 'center',
+    margin: 0,
+    width: '80%',
+    height: 50,
+  },
+  DisabledColor: {
+    backgroundColor: '#cccccc',
+  },
+  LoginScreenContainer: {
+    padding: 20,
+    flex: 1,
+  },
+  hidden: {
+    opacity: 0,
+    height: 0,
+    width: 0,
+    maegin: 0,
+    padding: 0,
+  },
+  formHeader: {
+    color: '#5566dd',
+    fontSize: 24,
+    paddingTop: 20,
+    paddingBottom: 20,
+    fontWeight: 'bold',
+  },
+  alternateSignSeparator: {
+    paddingBottom: 10,
+    textAlign: 'center',
+  },
+  checkboxContainer: {
+    paddingVertical: 20,
+  },
+});
+
```
Can you call these styles in a separate file?

YilKanda commented on src/components/atoms/ToLoginMessage.js:
```diff
@@ -0,0 +1,18 @@
+
+export const ToLoginMessage = () => {
+  const goToLoginScreen = () => {
+    console.log('to login screen');
```
Do not upload console.log() code to GitHub, if you want to display a message I recommend using the [Alert](https://reactnative.dev/docs/alert) component.

Darkboy5120 committed:
Theme structure, severals styles split it

- A themes structure were added to implement future themes, and sort every color
- @YillKanda thanks for the feedback, the requested changes were fixed, the ttf, the styles imports and the alert intead the console.log were implement too.

Co-Authored-By: Nando San Miguel Lopez <contact@namoku.xyz>
Co-Authored-By: thisisofficial <thisisofficiallyofficial@gmail.com>

Darkboy5120 committed:
Code refactorization

- LoginScreen componentes split then in new molecules
- foo foo

Co-Authored-By: Nando San Miguel Lopez <contact@namoku.xyz>
Co-Authored-By: thisisofficial <thisisofficiallyofficial@gmail.com>

YilKanda commented on src/theme/light.js:
```diff
@@ -0,0 +1,20 @@
+import {StyleSheet} from 'react-native';
+
+/*const Colors = {
+  primaryColor: '#5566dd',
+  disabledColor: '#cccccc',
+};*/
```
Try not to leave commented code on GitHub. If you are not using it, it is better to delete it.

Namoku replied:
Done, I've already deleted.

Commit:
Delete comments

Namoku reviewed (commented):
resolved

Namoku reviewed (commented):
done

DemisRincon reviewed (approved):
It's ok, but the imports of the styles could be indexed (exported all from an index) as your molecules component, in general, the code is o but keep your code structure consistent index every export or remove the indexed structure
